### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by gh-matv**

### CODE REVIEW

**Comment by gh-matv** [comment]
test

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# KitWorld
```
Comment (nicolasventer) [inline README.md:1]: bla

**Review by nicolasventer — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+## KitWorld
```
Comment (nicolasventer) [inline README.md:2]: another comment

Comment (gh-matv) [reply README.md:2]: ok thanks

**Review by nicolasventer — APPROVED** [review]

**Pull request merged**